chat-widget/src/common/telemetry/TelemetryHelper.ts

```diff
@@ -335,4 +335,54 @@ export class TelemetryHelper {
         };
         BroadcastService.postMessage(telemetryEvent);
     }
+
+      public static logLoadingEventToAllTelemetry = (logLevel: LogLevel, payload: TelemetryEventWrapper) => {
```

Comment:
looks like the same code is repeated multiple times , which contains small differences among them,  please evaluate to abstract it